app/views/discovered_hosts/_discovered_host.html.erb

```diff
@@ -5,6 +5,10 @@
 <td class="hidden-tablet hidden-xs"><%= number_to_human_size(discovery_attribute(host, :memory, 0) * 1024 * 1024) %></td>
 <td class="hidden-tablet hidden-xs"><%= discovery_attribute(host, :disk_count) %></td>
 <td class="hidden-tablet hidden-xs"><%= number_to_human_size(discovery_attribute(host, :disks_size, 0) * 1024 * 1024) %></td>
-<% Setting::Discovered.discovery_fact_column_array.each do |fact_column| %>
-<td class="hidden-tablet hidden-xs"><%= host.facts_hash[fact_column.strip] || 'N/A' %></td>
+<%
+  # can be replaced with pluck in Rails 4+
```

Comment:
Strange, I had to mess up rebasing, this was already replaced with `host.fact_values.joins(:fact_name).where('fact_names.name' => Setting::Discovered.discovery_fact_column_array).pluck(:name, :value)`. This is the old version from Rail 3 time. Fixed, pushed, also renamed `host` above. I also moved the line to controller, which is a better fit.